### ISSUE TITLE
Unable to pair Tradfri led driver

### ISSUE THREAD

**Issue by jmijnhout** [Koenkk/zigbee2mqtt#13139]
### What happened?

I tried all night to pair this device to my Zigbee2mqtt network:
https://www.zigbee2mqtt.io/devices/ICPSHC24-10EU-IL-1.html

I double checked, i have this exact device. After searching the internet, i found lots of people having trouble with these things, but no issues when connecting to Hue or the official Tradfri gateway. I use a Sonoff Zigbee 3.0 stick, but it finds nothing. Then i tried my old CC2531 stick, which got to the part of finding a device, but never completed the interview. This only works when i put the CC2531 on top of the driver.

1. Apparently you can set a Hue gateway in "search mode". Is this the same as "Allow devices to join" in Zigbee2mqtt?
2. How do you handle close proximity for the Sonoff Zigbee 3.0 stick? It has an antenna, and afaik this creates some kind of "donut" around it as a reception area. This means, putting it next to a device, would actually have a negative effect on the signal. How am i supposed to hold this close to the driver?

### What did you expect to happen?

The driver should be reconized by Zigbee2mqt, using the steps described here:
https://www.zigbee2mqtt.io/devices/ICPSHC24-10EU-IL-1.html

However, single pressing the button seems to not have worked for anyone. Keeping the button pressed for about 10 seconds allows me to get into an "interview" state every time with the CC2531, but never with the Sonoff Zigbee 3.0 stick (probably the signal issue). 

### How to reproduce it (minimal and precise)

CC2531:

1. Keep Zigbee stick on top of Led driver
2. Keep the led driver reset button pressed for ~10 seconds
3. Zigbee2mqtt sees a device announced
4. Zigbee2mqtt start interview
5. ... nothing after that...

Sonoff Zigbee 3.0 stick:

1. Keep Zigbee stick on top/bottom/side of Led driver / few inches away / long distance away / without antenna / with antenna
2. Keep the led driver reset button pressed for ~10 seconds
3. ... nothing ...

### Zigbee2MQTT version

1.26.0

### Adapter firmware version

20220219

### Adapter

Sonoff Zigbee 3.0 (+ CC2531 as test device)

### Debug log

(only on CC2531)

```
info  2022-07-13 22:28:25: Device '0x680ae2fffe3b415d' joined
info  2022-07-13 22:28:25: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x680ae2fffe3b415d","ieee_address":"0x680ae2fffe3b415d"},"type":"device_joined"}'
info  2022-07-13 22:28:25: Starting interview of '0x680ae2fffe3b415d'
info  2022-07-13 22:28:25: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x680ae2fffe3b415d","ieee_address":"0x680ae2fffe3b415d","status":"started"},"type":"device_interview"}'
```

... nothing after this

**Comment by jmijnhout** [comment]
Ok i solved the issue last night. It was a big effort, as i need to repair all my devices, but at least now everything works. Here's my story:

I read on several forums that users who were experiencing trouble pairing these devices, blamed the channel they were on. Some were on channel 24, and switched to channel 14, and stuff started working. Since i was on channel 11, which is one of the recommended channels, i never suspected this to impact me. However, i did a small test with the CC2531, and guess what, it paired instantly! I didn't even have to hold it close to the coordinator! Then i tried channel 15, which is more recommended. Also no issues here, instant pair.

The pairing description on the product page of Zigbee2mqtt is wrong though (https://www.zigbee2mqtt.io/devices/ICPSHC24-10EU-IL-1.html). As you can also find in some Youtube videos, showing users pairing this device to a Hue bridge. You need to keep the reset button **pressed for several seconds**. A single press, as described, will not work. If you have a light attached to it, it will dim and blink once. Keep it pressed longer until the green "pairing" popups start showing. Release the button, and Zigbee2mqtt will do the rest.

So next step, set my main network with the Sonoff Zigbee 3.0 stick to channel 15, and see if it works. I was about 5m from the coordinator with a brick wall in between, but it detected the led drivers immediately, and several seconds later the led driver was paired successfully. After that i need to repair all my devices bc of the channel switch though, but now everything works.

TL;DR

1. The pairing description here: https://www.zigbee2mqtt.io/devices/ICPSHC24-10EU-IL-1.html is incorrect. You don't need to keep the device close to the coordinator, and you need to keep the button pressed until pairing starts.
2. These led drivers are very picky on which channel you are. Apparently channel 24 is a nogo, proven by others, and channel 11 also is off limits, proven by me.
